embeds/stats_embeds.py

```diff
@@ -11,14 +11,18 @@
 
 
 @to_thread
-def stats_embed(leetcode_username: str, display_name: str) -> Tuple[discord.Embed, discord.File | None]:
+def stats_embed(leetcode_username: str, display_name: str, heatmap: bool) -> Tuple[discord.Embed, discord.File | None]:
```

Comment:
I'd recommend this parameter specifically to be called `extension` and by default set to None. If None, don't display an extension, if its "activity" or "heatmap" display those extensions. So, in cogs/stats.py, you can check if heatmap variable is False, if so pass to stats_embed "activity", if not then pass "heatmap". This is just so to make it easier to expand this capability and allow user's to pick a different extension (cos I believe there are 3 extensions in total). 

Although ideally the `heatmap` param in cogs/stats would better to be suited to be an optional choice of the 3 strings, but that's something I need to implement everywhere at some point in time.